app/controllers/update_username_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_dependency "#{Rails.root}/lib/importers/user_importer"
+
+class UpdateUsernameController < ApplicationController
+  before_action :require_signed_in
+  respond_to :html, :json
+  def index
+    @user = User
```

Comment:
I don't think this does anything.

Reply:
Yes but do i need to initialize anything in the index method(controller)? To make the page render?

Reply:
I'll probably try to remove it and see if everything works properly. Else @ragesoss just let me know where I need to add link to this page?

Reply:
You should keep the index method, it just doesn't need to set any instance variables. `@user = User` is just setting the instance variable `@user` to point to the class `User` (rather than an instance of that class). That's almost never something we'd want to do, as it's likely to cause confusion; everywhere else in the codebase, an instance variable with that name would be used for a single user record, not the class itself.

Reply:
Ok I'll just keep empty method

Reply:
I think the only place that should link to this page would be the `/admin` index, in the 'Useful links' list. We'll add another link from an FAQ, but that content is not part of the codebase itself.

Reply:
Okk